sechub-developertools/scripts/container/postgres/run.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env sh
+# SPDX-License-Identifier: MIT
+
+debug () {
+    while true
+    do
+	    echo "Press [CTRL+C] to stop.."
+	    sleep 5s
+    done
+}
+
+server () {
+    # start PostgreSQL server
+    pg_ctl start
+
+    psql --command="CREATE USER $POSTGRES_DB_USER PASSWORD '$POSTGRES_DB_PASSWORD';"
+    psql --command="CREATE DATABASE $POSTGRES_DB_NAME OWNER $POSTGRES_DB_USER;"
+
+    # check PostgreSQL server status,
+    # for the container to stay alive
+    while pg_ctl status
+    do
+        sleep 5m
```

Comment:
This might be a little long. I would prefer 1-2 minutes.

Reply:
Set to 1m, DONE